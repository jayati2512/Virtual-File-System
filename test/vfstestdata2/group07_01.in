createvfs root/vfs 1000
createvfs vfs 2000
createvfs vfs 1000
mountvfs vfs1
mountvfs vfs
listdir / 1  /tmp/out.txt
addfile stdio.h /user/include/stduio.h
listfile input.txt /tmp/
unmountvfs
