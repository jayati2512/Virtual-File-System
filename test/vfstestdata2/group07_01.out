INVALID_CHARACTER_IN_NAME
INVALID_SIZE
success
DATA_FILE_NOT_FOUND
success
success
VFS_INSUFFICIENT_ARGUMENTS
SOURCE_FILE_PATH_NOT_FOUND
VFS_INSUFFICIENT_ARGUMENTS
